exercicios/exercicio4.js

```diff
@@ -20,3 +20,14 @@ const listaDeMercado = [
     comprado: false
   }
 ]
+
+// 4. Filtre os itens já comprados de uma lista de mercado e retorne a lista atualizada
+
+// console.log(listaDeMercado)
+
+for (let i = 0; i<listaDeMercado.length; i++) {
```

Comment:
Gostei da sua solução usando o `for`! E se a gente usasse os métodos de array do Javascript?